src/core/Form/TextInput/TextInput.md

```diff
@@ -101,5 +101,11 @@ import { TextInput } from 'suomifi-ui-components';
     icon="mapLocation"
     iconProps={{ fill: 'red' }}
   />
+  <TextInput
+    onBlur={(event) => console.log(event.target.value)}
+    labelText="TextInput with debounced onChange event"
+    onChange={(value) => console.log(value)}
```

Comment:
Should we have both onBlur and onChange applied to the same example?

Reply:
Moved the onBlur to another example.